mautic/mautic
# Change ajax call on keyup instead of enter key integration

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | No
| Automated tests included? | No
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Change search request on key up instead of enter key.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Go To - Plugins -> Salesforce -> Lead Field Mapping (Configure salesforce if not exists )
use  for configure salesforce.
2. In Filter user need to press "Enter" to filter field.
3. Switch to "Lead Field Mapping" tab
4. Start typing in "Filter Integration fields"
5. Type word with space Ex. "Last Name" or "Clean S".
6. The fields are available but would not display fields.


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Go To - Plugins -> Salesforce -> Lead Field Mapping 
3. Field should be filter on type keyword only.
4. Switch to "Lead Field Mapping" tab
5. Start typing in "Filter Integration fields"
6. Type word with space Ex. "Last Name" or "Clean S".
7. If fields are available then should be display.

#### Other areas of Mautic that may be affected by the change:
1. 
2. 

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
